modules/understanding-insights-operator-alerts.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-operator.adoc
+
+
+:_content-type: CONCEPT
+[id="understanding-insights-operator-alerts_{context}"]
+= Understanding Insights Operator alerts
+
+Insights Operator declares alerts through the Prometheus monitoring system to Alertmanager. You can view these alerts in the Alerting UI accessible through the Administrator perspective and the Developer perspective in the {product-title} web console. 
+
+Currently, Insights Operator will send the following alerts when the conditions are met:
+
+.Insights Operator alerts
+[options="header"]
+|====
+|Alert|Description
+|`InsightsDisabled`|Insights operator is disabled.
```

Comment:
Per OpenShift doc guidelines, the term ["Operator" is always capitalized](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-capitalization) 
```suggestion
|`InsightsDisabled`|Insights Operator is disabled.
```